gpu_ocean/SWESimulators/CDKLM16.py

```diff
@@ -125,24 +127,26 @@ def __init__(self, \
         ghost_cells_x = 2
         ghost_cells_y = 2
         
-        #Coriolis at "first" cell
-        x_zero_reference_cell = ghost_cells_x
-        y_zero_reference_cell = ghost_cells_y # In order to pass it to the super constructor
```

Comment:
Dette må være med for å hoppe over ghostcellene. Slik du har implementert starter coriolis_x0 fra inklusivt ghost-cellene, re vår samtale på kontoret i dag.

Reply:
Nei - Dette er grunnen til at det er minus to overalt i kjernen.